libraries/Hash/src/Hash.h

```diff
@@ -31,12 +31,12 @@ void sha1(uint8_t * data, uint32_t size, uint8_t hash[20]);
 void sha1(char * data, uint32_t size, uint8_t hash[20]);
 void sha1(const uint8_t * data, uint32_t size, uint8_t hash[20]);
 void sha1(const char * data, uint32_t size, uint8_t hash[20]);
-void sha1(String data, uint8_t hash[20]);
+void sha1(const String& data, uint8_t hash[20]);

```

Comment:
Can you add the `const` modifier to the `uint8_t *` prototype on line 30 and line 36 as well?  They don't copy anything, but they also are constant input parameters anyway.

Reply:
Will do